bin/node/runtime/src/lib.rs

```diff
@@ -667,7 +667,11 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type RewardHandler = (); // nothing to do upon rewards
 	type DataProvider = Staking;
 	type Solution = NposSolution16;
-	type Fallback = pallet_election_provider_multi_phase::NoFallback<Self>;
+	type Fallback = frame_election_provider_support::onchain::BoundedOnChainSequentialPhragmen<
+		Self,
+		frame_support::traits::ConstU32<10>,
+		frame_support::traits::ConstU32<100_000>,
+	>;
```

Comment:
At this point I think it is good to import these types to make the line a bit shorter.